spring-cloud-starter-tencent-polaris-router/src/main/java/com/tencent/cloud/polaris/router/config/RouterAutoConfiguration.java

```diff
@@ -85,4 +95,19 @@ public NearbyRouterRequestInterceptor nearbyRouterRequestInterceptor(PolarisNear
 	public RuleBasedRouterRequestInterceptor ruleBasedRouterRequestInterceptor(PolarisRuleBasedRouterProperties polarisRuleBasedRouterProperties) {
 		return new RuleBasedRouterRequestInterceptor(polarisRuleBasedRouterProperties);
 	}
+
+	@SuppressWarnings("rawtypes")
+	@Autowired(required = false)
+	private List<RibbonRequestCustomizer> requestCustomizers = Collections.emptyList();
```

Comment:
> Autowired `List<RibbonRequestCustomizer>` property can moved into `PolarisRibbonRoutingFilter` initialize method ?

You mean use `BeanFactoryUtils` in `PolarisRibbonRoutingFilter` constructor instead of `@Autowired`?

Reply:
Using `@Autowired` annotation in `AutoConfiguration` Class is inelegant ~

```java
@Bean
@ConditionalOnClass(name = "org.springframework.cloud.netflix.zuul.filters.route.RibbonRoutingFilter")
public PolarisRibbonRoutingFilter ribbonRoutingFilter(ProxyRequestHelper helper,
		RibbonCommandFactory<?> ribbonCommandFactory,
		MetadataLocalProperties metadataLocalProperties,
		RouterRuleLabelResolver routerRuleLabelResolver,
		@Nullable List<RibbonRequestCustomizer> requestCustomizers,
		List<ServletRouterLabelResolver> routerLabelResolvers) {
	return new PolarisRibbonRoutingFilter(helper, ribbonCommandFactory,
			requestCustomizers, metadataLocalProperties, routerRuleLabelResolver, routerLabelResolvers);
}
```